src/Tests/PlayerLogicTest.java

```diff
@@ -44,4 +44,56 @@ public void testPlacedPieceLocationIsCorrect() {
       assertSame(playerLogic.getBoardAsPlayerTokens()[2], PlayerToken.PLAYER1);
       assertSame(playerLogic.getBoardAsPlayerTokens()[6], PlayerToken.PLAYER2);
   }
+
+  /***** SPRINT 2 TESTS *****/
+  public void testP1IndexOccupied() {   //player 1 move invalid when occupied
+      playerLogic.placePlayerPiece(5);
+      playerLogic.placePlayerPiece(4);
+      assertFalse(playerLogic.move(4, 5));
+  }
+    public void testP2IndexOccupied() {   //player 1 move invalid when occupied
+        playerLogic.nextTurn();
+        playerLogic.placePlayerPiece(5);
+        playerLogic.placePlayerPiece(4);
+        assertFalse(playerLogic.move(4, 5));
+    }
```

Comment:
Maybe go ahead and make these one test function called `testCannotMoveWhenDestinationOccupied`